path_test.go

```diff
@@ -125,6 +125,30 @@ func Test_zoneFromPath(t *testing.T) {
 			"_redirect.string.example.com",
 			nil,
 		},
+		{
+			"example.com",
+			"/12345-some-path?query=string&more=stuff",
+			"",
+			"\\?query=(?P<a>[^&]+)\\&more=(?P<b>[^&]+)",
+			"_redirect.string.stuff.example.com",
+			nil,
+		},
+		{
+			"example.com",
+			"/12345-some-path?query=string&more=stuff&test=testing",
+			"",
+			"\\?query=(?P<b>[^&]+)\\&more=(?P<a>[^&]+)\\&test=(?P<c>[^&]+)",
+			"_redirect.stuff.string.testing.example.com",
```

Comment:
Shouldn't the ordering be:
```
"_redirect.testing.string.stuff"
```
So the query matches `string` as `b`, `stuff` as `a` and `testing` as `c`.
Using our usual ordering would mean _redirect.c.b.a.example.com.
Am I missing something?

Reply:
Forgot to use ``reverse()``. Changed the tests and used reverse in the last commit.